docs/documentation/tooling_documentation.md

```diff
@@ -0,0 +1,24 @@
+# Please note that tooling is meant to be for internal use only.
+
+# What it does
+The tooling programmatically generates the images required for the animation process by looping through layers and 
+toggling visibility.
+
+# Usage
+This script can be run by going to Illustrator -> File -> Scripts -> Browse & selecting this script. you will also have
```

Comment:
I recommend writing it in order. Change the file path first, then go into Illustrator and select the script. The file path can be anywhere they want, but we recommend creating their own folder because it generates A LOT of images. After running the script, wait for the script to finish. Illustrator will take a while and might show that it's not responding, but the user will just have to wait a couple minutes and it'll be fine.

Reply:
That's a great idea :) I'll get that updated in the next iteration when we add the images and things!